content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions.md

```diff
@@ -184,9 +184,9 @@ jobs:
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
       # The following properties are now available:
-      #  - steps.metadata.outputs.dependency-names
-      #  - steps.metadata.outputs.dependency-type
-      #  - steps.metadata.outputs.update-type      
+      #  - steps.dependabot-metadata.outputs.dependency-names
```

Comment:
This line refers to the `id` of the previous step on line 182, which is called `metadata` in this example. In the [dependabot/fetch-metadata README](https://github.com/dependabot/fetch-metadata/blob/main/README.md?plain=1#L94) this step is called `dependabot-metadata` which is why the two examples have diverged.

I think being consistent and using `dependabot-metadata` in both places is worthwhile, but we should also update line 182.